readme.md

```diff
@@ -34,7 +35,9 @@ Elements in beautiful READMEs include, but not limited to: images, screenshots,
 - ["Art of Readme - Learn the art of writing quality READMEs."](https://github.com/noffle/art-of-readme) - *Stephen Whitmore*
 
 ## Tools
-- [Standard Readme](https://github.com/RichardLitt/standard-readme) - A standard readme style specification. Has a generator to help create spec-compliant readmes, too.
+
+- [Standard Readme](https://github.com/RichardLitt/standard-readme) - A standard README style specification. Has a generator to help creat spec-compliant READMEs, too.
+- [Zalando's README Template](https://github.com/zalando/zalando-howto-open-source/blob/master/READMEtemplate.md) - Simple template to help you cover all the basics.
```

Comment:
@RichardLitt sure. My colleague shared the link to maintainer.io with me a couple weeks ago; I thought about reaching out to you guys but got busy flying around Europe. And now here we are. :) I wonder if there is some way we could collaborate or help each other.